lib/measured/measurable.rb

```diff
@@ -9,7 +9,7 @@ def initialize(value, unit)
     @unit = unit_from_unit_or_name!(unit)
     @value = case value
     when Float
-      BigDecimal(value, Float::DIG + 1)
+      value.to_d
```

Comment:
`to_d` requires people using this project to `require 'bigdecimal/util'`, which we don't want to impose. I'd rather just switch to `Float::DIG` instead of `Float::DIG + 1`.

Reply:
Good call, thanks :)